proposals/wac-ucr/index.bs

```diff
@@ -846,6 +846,7 @@ with conditions that stipulate:
 * she can only create or change [=authorizations=]
     targeting the `measurements` [=collection=] where
     [=agents=] are included in the `interns` [=authorization group=].
+* she can't extend or transfer [=control access=] to anyone else.
```

Comment:
I think this bullet point could harmonize better with a previous one. I understand that that Megan should be able to grant *read* or *read-write* access to any members of *interns* group. She can't modify here own *restricted control access* or grant any degree of *restricted control access* to anyone else.

Reply:
The previous point is already borderline convoluted. I kept the points separate to help distinguish the intentions. Feel free to update though. This whole use case needs to be re-written.